gnucash/report/reports/example/sample-report.scm

```diff
@@ -1,5 +1,8 @@
 ;; -*-scheme-*-
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; sample-report.scm : Options Example Report
+;; refactored from hello-world by flywire January 2023

```

Comment:
I understand that you're trying to protect your identity here but copyright is a legal thing so to make a useful assertion of it on your changes you must use your real legal name. If you don't care to assert copyright then what you did and when will be preserved in the git history and  adding a comment isn't necessary.

Reply:
The intent is to explicitly mention hello-world for users that find references to it but do not check git history. This form seems consistent with comments about major changes in other reports but I'll take advice.

---

I'm not concerned about copyright within GnuCash, although I understand under a pseudonym it expires 70 years after publication rather than the author's death. Maybe that's only Australia.

Reply:
I believe that's international, as most copyright law is these days (but IANAL). That purpose is served by the git history, which is why the comment isn't necessary.

If you `git mv` hello-world.scm->sample-report.scm then that purpose is also served by git history.

Reply:
Text deleted.